aptos-move/e2e-tests/src/executor.rs

```diff
@@ -122,11 +118,6 @@ impl FakeExecutor {
         Self::from_genesis(GENESIS_CHANGE_SET_TESTNET.clone().write_set())
     }
 
-    /// Creates an executor using the mainnet genesis.
-    pub fn from_mainnet_genesis() -> Self {
```

Comment:
Outdated mainnet genesis code that has been replaced with a different logic now.